Code/GraphMol/DistGeomHelpers/Embedder.h

```diff
@@ -120,6 +109,10 @@ struct RDKIT_DISTGEOMHELPERS_EXPORT EmbedParameters {
   unsigned int ETversion;
   boost::shared_ptr<const DistGeom::BoundsMatrix> boundsMat;
   bool embedFragmentsSeparately;
+  bool useSmallRingTorsions;
+  bool useMacrocycleTorsions;
+  bool useMacrocycle14config;
+  const std::map<std::pair<unsigned int, unsigned int>, double> *CPCI;
```

Comment:
The ownership of this pointer is not clear, which inevitably will lead to memory leaks. 
It's safest to make it an `std::shared_ptr<std::map<std::pair<unsigned int, unsigned int>, double>>`